asserts/internal/grouping.go

```diff
@@ -169,13 +221,52 @@ func (gr *Groupings) Deserialize(label string) (*Grouping, error) {
 	return &g, nil
 }
 
+func (gr *Groupings) bitsetDeserialize(g *Grouping, b []byte) (*Grouping, error) {
+	buf := bytes.NewBuffer(b)
+	binary.Read(buf, binary.LittleEndian, &g.size)
+	if g.size > gr.maxGroup+1 {
+		return nil, errSerializedLabel
+	}
+	g.elems = make([]uint16, gr.bitsetThreshold)
+	binary.Read(buf, binary.LittleEndian, g.elems)
```

Comment:
Missing error check.

Reply:
this can only fail if b is too small but size is checked in the caller to be exactly right